apps/console/src/features/connections/constants/autheticator-constants.ts

```diff
@@ -366,4 +366,6 @@ export class AuthenticatorManagementConstants {
             "There are applications using this connection."
         )
     }
+
+    public static readonly scimId: string = "c2NpbQ";
```

Comment:
Shall we use something like `DEPRECATED_SCIM1.0_CONNECTOR`?

Reply:
Improved with 717834fb9a01b8742370af47d8374ff5081fa8d1